e2e-tests/version-service/conf/version-service-exact-rs0.yml

```diff
@@ -5,6 +5,7 @@ metadata:
   finalizers:
     - delete-psmdb-pvc
 spec:
+  crVersion: 9.9.9
   image:
```

Comment:
So we going to use 9.9.9 version? I will fix it in my PR in this case.

Reply:
@egegunes

Reply:
yes, please fix